scripts/demo/demo-magic/demo-magic.sh

```diff
@@ -25,13 +25,18 @@ SHOW_CMD_NUMS=false
 
```

Comment:
This is third-party software, shouldn't modify it

Reply:
Done